src/utils.js

```diff
@@ -572,7 +572,16 @@ export default {
       let i
       let nu
 
-      if (typeof value === 'object' && value !== null && !(value instanceof Boolean) && !(value instanceof Date) && !(value instanceof Number) && !(value instanceof RegExp) && !(value instanceof String) && !(value instanceof File)) {
+      if (
+        typeof value === 'object' &&
+        value !== null &&
+        !(value instanceof Boolean) &&
+        !(value instanceof Date) &&
+        !(value instanceof Number) &&
+        !(value instanceof RegExp) &&
+        !(value instanceof String) &&
+        (typeof File === 'undefined' || !(value instanceof File))
```

Comment:
Just stylistic preference, I can update the PR to just be !File if you'd prefer?

> On 23 Oct 2015, at 16:58, Jason Dobry <notifications@github.com> wrote:
> 
> In src/utils.js:
> 
> > @@ -572,7 +572,16 @@ export default {
> >        let i
> >        let nu
> >  
> > -      if (typeof value === 'object' && value !== null && !(value instanceof Boolean) && !(value instanceof Date) && !(value instanceof Number) && !(value instanceof RegExp) && !(value instanceof String) && !(value instanceof File)) {
> > +      if (
> > +        typeof value === 'object' &&
> > +        value !== null &&
> > +        !(value instanceof Boolean) &&
> > +        !(value instanceof Date) &&
> > +        !(value instanceof Number) &&
> > +        !(value instanceof RegExp) &&
> > +        !(value instanceof String) &&
> > +        (typeof File === 'undefined' || !(value instanceof File))
> Instead of typeof File === 'undefined' why not just do !File?
> 
> —
> Reply to this email directly or view it on GitHub.
>